packages/@aws-cdk/aws-pipes-targets-alpha/README.md

```diff
@@ -122,3 +123,51 @@ const pipe = new pipes.Pipe(this, 'Pipe', {
     target: pipeTarget
 });
 ```
+
+### AWS Lambda Function
+
+A Lambda Function can be used as a target for a pipe. The Lambda Function will be invoked with the (enriched/filtered) source payload.
```

Comment:
Can you explain what you meant by `(enriched/filtered) source payload`?

Reply:
I meant the enrichment step of the pipe. I removed the word filtered as its wrong. Thanks for the find